scripts/check-copyright

```diff
@@ -0,0 +1,11 @@
+files=`find . -path './tpls' -prune -o -name '*.cpp' -o -name '*.pc.in' -o -name '*.hpp' -o -name '*.cc'`
```

Comment:
```
cmake/compile_tests/cuda_compute_capability.cc
tpls/gtest/gtest/gtest-all.cc
```
I am tempted to say let us just rename `cuda_compute_capability.{cc -> cpp}` and drop the `*.cc`

Also why `*.pc.in`?

Reply:
not sure, was in the file from before.